project-base/storefront/components/Pages/Search/helpers.ts

```diff
@@ -27,23 +25,12 @@ export const useSearchProductsData = (
     const previousPageRef = useRef(currentPage);
     const initialPageSizeRef = useRef(calculatePageSize(currentLoadMore));
 
-    const userIdentifier = usePersistStore((store) => store.userId)!;
-
-    const [searchProductsData, setSearchProductsData] = useState(
-        readSearchProductsFromCache(
-            client,
-            searchString ?? '',
-            sort,
-            mappedFilter,
-            getEndCursor(currentPage),
-            initialPageSizeRef.current,
-            userIdentifier,
-        ),
-    );
-
```

Comment:
Just so you know and realize it, this means that there will be no data initially. I know we talked about making this page CSR, so then its okay, but just so it has been said.

Reply:
Yes, this is correct, we don't have anymore SSR on this page. That's why we have additional skeletons.